pkg/application/app.go

```diff
@@ -187,6 +187,10 @@ func (app *Avalanche) listSideCarNames() ([]string, error) {
 	return names, nil
 }
 
+func (app *Avalanche) GetSidecarNames() ([]string, error) {
```

Comment:
nit: Do we want to just make `listSideCarNames` visible externally instead of wrapping it?

Reply:
OK agreed.